crates/vizia_core/src/views/slider.rs

```diff
@@ -197,6 +201,10 @@ impl<L: Lens> View for Slider<L> {
             SliderEventInternal::SetRange(range) => {
                 self.internal.range = range.clone();
             }
+
+            SliderEventInternal::SetKeyboardFraction(keyboard_fraction) => {
+                self.internal.keyboard_fraction = keyboard_fraction;
```

Comment:
Compiler is complaining that `keyboard_fraction` needs dereferencing here.

Reply:
Wasn't an error on whatever version of rust I used to be using but it is on the current stable.